testeng/resources/create-repo-health-report.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e -v
+
+# click requires this to work cause it interfaces weirdly with python 3 ASCII default
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+virtualenv --python=/usr/bin/python3.6 venv -q
+source venv/bin/activate
+
+ls
```

Comment:
I guess there is no significance of this `ls` here? added for testing maybe?

Reply:
yep, its added for testing, will remove